AI_ML/CLASSIFICATION/ONE_HOT_ENCODING/ONE_HOT_ENCODING.py

```diff
@@ -1,10 +1,15 @@
-from flojoy import flojoy, DataContainer
+from flojoy import flojoy, Matrix, DataFrame, Array
+
+from typing import Optional
 import pandas as pd
-from typing import cast
 
 
 @flojoy
-def ONE_HOT_ENCODING(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
+def ONE_HOT_ENCODING(
+    default: DataFrame,
```

Comment:
Why not just support only dataframes?
I feel supporting "categories" and/or "columns" makes thise node more complicated and error prone.

Reply:
That's a good point. I was actually running into errors when handling categories. However, I think columns params should be supported so that users can freely select any columns from data to be turned into OHE.

Reply:
I agree 100% and it's ok to have that in one node unless you deem it unnecessary to have that "column selection" functionality be its own separate node (i.e. for users to select the columns they need, they can compose an "EXTRACT_COLUMN" node + "ONE_HOT_ENCODING" node). Please feel free to proceed with whichever option.

Reply:
That's also valid, in that case then I would need a second input which contains the extracted columns on which OHE can be performed. However, I just find it a bit uncomfortable for users, if they end up selecting all columns to yield OHE, then that process might be cumbersome.

Reply:
Therefore, I kept the columns option

Reply:
I made the changes!